src/main/java/pl/javastart/task/Brutto.java

```diff
@@ -0,0 +1,25 @@
+package pl.javastart.task;
+
+public class Brutto {
+    double priceBrutto(Product product) {
+        double price;
+        if (product.getCategory() == null) {
+            price=product.getPrice() * 0.03 + product.getPrice();
```

Comment:
Checkstyle znalazł w tej linii błąd. Polecam skonfigurować sobie dodatek w IntelliJ, żeby na bieżąco śledził problemy. Tutaj instrukcja jak to zrobić: https://bootcamp.javastart.pl/strona/jakosc-kodu

Na tej stronie https://github.com/javastartbootcamp/java-alior_arokosandria_zad_4.3/pull/1/files widać wszystkie komentarze, który dodał checkstyle (mamy automat po każdym commicie na githuba)